### ISSUE TITLE
update perf_predict to use gperf heap profilier

### ISSUE THREAD

**Issue by grahamlopez** [project-asgard/asgard#93]
Skip to content

ASGarD
    asgardasgard
    Issues
    #93

Open
Opened 4 days ago by Graham Lopez
Close issue
New issue
update perf_predict to use gperf heap profilier

There is a possibility that gperf's heap profiler would be faster than waiting on valgrind. Let's make it a user option for which one to use.
Related issues
0

Graham Lopez @bgl added infra performance labels 4 days ago

**Comment by grahamlopez** [comment]
closing as we are not able to actively maintain perfpredict for right now
